parachain/runtime/runtime-tests/src/relaychain/kusama_cross_chain_transfer.rs

```diff
@@ -57,7 +57,7 @@ fn transfer_to_relay_chain() {
     });
 
     KusamaNet::execute_with(|| {
-        let xcm_fee = 106_666_660;
+        let xcm_fee = 165_940_672;
```

Comment:
Where did this magic number come from? Will this test break on upgrade and if so how do we change this?

Reply:
The fee calculations changed on kusama: https://github.com/paritytech/polkadot/pull/5202, in particular the `ExtrinsicBaseWeight`. See https://github.com/interlay/interbtc-ui/pull/288 for more detail

Reply:
@sander2 can you add that as a comment? Will make it easier to upgrade in the future.